testing-examples/naming-runs/naming-test-runs.md

```diff
@@ -0,0 +1,15 @@
+# Want to name your test runs? Here's how!
+You can easily name langsmith test runs by setting the `project_name` parameter. We recommend including a string identifier along with a time-based component so that you can easily tell your test runs apart, e.g."\<name of test\> {timestamp}". For example:
+
+```python
+project_name = f"improved_chain {datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')}"
+
+results = client.run_on_dataset(
+    project_name=project_name,
```

Comment:
We should put `...` above the project name since there are other required args (this is optional, so would follow them)